x-pack/plugins/enterprise_search/public/applications/enterprise_search_overview/components/product_selector/product_selector.tsx

```diff
@@ -62,11 +62,11 @@ export const ProductSelector: React.FC = () => {
           <ApiKeyPanel />
           <EuiSpacer size="xl" />
           <EuiTitle>
-            <h4>
+            <h2>
               {i18n.translate('xpack.enterpriseSearch.productSelector.overview.title', {
                 defaultMessage: 'Ingest your content',
               })}
-            </h4>
+            </h2>
```

Comment:
If we change to `h2`, should we update other headers(such as `Create a custom search experience` on the same page too)?

Reply:
Possibly. We tend to use `hX` tags to indicate a specific size, but we should be using them indicate a hierarchy of importance. That is: an `h1` should be the most important header on the page. An `h2` should always be nested under an `h1` somewhere etc. We should improve this, but it's manual work and checking every time is difficult.